firmware/applications/SignBuddy/SignBuddy.proto

```diff
@@ -9,9 +9,9 @@ message Sample {
     required uint32 sample_id = 1;
 
     message IMUData {
-        required sint32 eul_p = 1;
+        required sint32 eul_h = 1;
         required sint32 eul_r = 2;
-        required sint32 eul_y = 3;
+        required sint32 eul_p = 3;
         required sint32 lin_acc_x = 4;
         required sint32 lin_acc_y = 5;
```

Comment:
Need to regenerate proto header and source

Reply:
make proto didn't work for me. Am i supposed to do something else before executing it?

Reply:
You might need to rebuild the docker image if you haven't done that since we added proto, if that doesn't work can you send me the error you are getting?